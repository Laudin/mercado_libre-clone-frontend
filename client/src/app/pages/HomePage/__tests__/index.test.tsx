import * as React from 'react'
import { createRenderer } from 'react-test-renderer/shallow'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { HomePage } from '../index'

const shallowRenderer = createRenderer()

describe('<HomePage />', () => {
   test('should render', () => {
      shallowRenderer.render(<HomePage />);
      const rendererOutput = shallowRenderer.getRenderOutput();
      expect(rendererOutput).toMatchSnapshot()
   })
})

//test styled-components
/* it('works fine', () => {
   const page = renderer.create(<HomePage />).toJSON()

}) */