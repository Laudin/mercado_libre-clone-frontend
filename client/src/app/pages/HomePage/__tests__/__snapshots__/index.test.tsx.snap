// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should render 1`] = `
<React.Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
    prioritizeSeoTags={false}
  >
    <title>
      Mercado Libre
    </title>
    <meta
      content=""
      name=""
    />
  </Helmet>
  <HomePage__Wrapper>
    <Slider />
    <HomePage__Content>
      <HomePage__Payments>
        <HomePage__Element>
          <HomePage__Image
            src=""
          />
          <HomePage__Title>
            Tarjeta de crédito
          </HomePage__Title>
        </HomePage__Element>
        <HomePage__Element>
          <HomePage__Image
            src=""
          />
          <HomePage__Title>
            Tarjeta de débito
          </HomePage__Title>
        </HomePage__Element>
        <HomePage__Element>
          <HomePage__Image
            src=""
          />
          <HomePage__Title>
            Cuotas sin tarjeta
          </HomePage__Title>
        </HomePage__Element>
        <HomePage__Element>
          <HomePage__Image
            src=""
          />
          <HomePage__Title>
            Efectivo
          </HomePage__Title>
        </HomePage__Element>
      </HomePage__Payments>
      <ProductsList
        category="Supermercado"
      />
      <ProductsList
        category="Electrodomésticos"
      />
      <ProductsList
        category="Vehículos"
      />
    </HomePage__Content>
  </HomePage__Wrapper>
</React.Fragment>
`;
